### PR TITLE
Reorganize README and Updating Getting Started Documentation

### PR DESCRIPTION
**Pull request by danielhelfand**
This pull request starts work on #233. This reorganizes the README to sooner calls out what prerequisites are needed for the examples and where Quarkus documentation is located. It also adds a `README` to the `getting-started` example that I hope will serve as an example for other examples under `quarkus-quickstarts`. 

Assuming this `README` approach works for examples, I can add ones to the other examples as well.

### CODE REVIEW

File: amqp-quickstart/pom.xml [hunk 1]
```diff
@@ -8,7 +8,7 @@
   <properties>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <surefire-plugin.version>2.22.0</surefire-plugin.version>
-    <quarkus.version>999-SNAPSHOT</quarkus.version>
+    <quarkus.version>0.18.0</quarkus.version>
```
Comment (gsmet) [inline amqp-quickstart/pom.xml]: This shouldn't be done. `development` needs to stay to `999-SNAPSHOT`. Please revert this part.

File: README.md [hunk 2]
```diff
@@ -2,6 +2,18 @@
 
 This repository contains a set of examples about the Quarkus framework.
 
+There is documentation published at <https://quarkus.io> (docs' [sources are here](https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc)).
```
Comment (gsmet) [inline README.md]:
```suggestion
The documentation for these quickstarts is published at <https://quarkus.io> (you can find [the documentation sources here](https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc)).
```

(while we're at it...)

File: amqp-quickstart/pom.xml (hunk 1 above)
Comment (danielhelfand) [reply amqp-quickstart/pom.xml]: Sorry, I originally branched off `master`. Will revert it.

File: getting-started/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+```
+./mvnw compile quarkus:dev
+```
+
+**Gradle**
+
+```
+./gradlew quarkusDev
+```
```
Comment (gsmet) [inline getting-started/README.md]: We don't really want this example to be built as a Gradle project. We have a PR separating the projects, I need to take a look at them.

I don't know if it's a good idea to add this file in each quickstart. I wonder if it won't be a nightmare to maintain.

Comment (danielhelfand) [reply getting-started/README.md]: I can agree to the issue of maintaining all these. Is there a general pattern to building all of these examples to just get them running? 

Should `./mvnw compile quarkus:dev` be enough to get everything up and running, or are there certain things that some examples will need in addition?

**Pull request closed without merging**